tools/js-sdk-release-tools/src/common/migration.ts

```diff
@@ -19,10 +20,16 @@ export async function migratePackage(packageDirectory: string): Promise<void> {
             cwd: packageDirectory,
         }
     );
-
-    logger.info(`Start to rush update after migration.`);
-    await runCommand(`node`, ['common/scripts/install-run-rush.js', 'update'], runCommandOptions, false);
-    logger.info(`Rush update successfully.`);
+    if (isRushRepo(sdkRepo)) {
+        logger.info(`Start to rush update after migration.`);
+        await runCommand(`node`, ['common/scripts/install-run-rush.js', 'update'], runCommandOptions, false);
+        logger.info(`Rush update successfully.`);
+    }else{
+        logger.info(`Start to install dependencies using pnpm after migration.`);
+        await runCommand(`pnpm`, ['install'], runCommandOptions, false);
```

Comment:
do we need to install pnpm package before invoking pnpm install?

Reply:
The pnpm package will be installed in [automation_init.sh](https://github.com/Azure/azure-sdk-for-js/pull/33993/commits/f7d4d26cca93c09f7f64147d6209c6b86eb58744#diff-48659f52f8b877bce024a10ede220148befe328d05834381607fb648ff695f3c).